src/constants/counter.js

```diff
@@ -0,0 +1,5 @@
+import { createConstants } from 'utils';
```

Comment:
Hey, could you change this to `../utils` (or whatever the correct relative path is). I know it works totally fine with the alias, but back when this starter kit came with a sample Todo app I received multiple comments asking to use a relative path for this purely for clarification since `utils` is kinda generic.